lms/djangoapps/courseware/tests/test_block_render.py

```diff
@@ -950,7 +950,7 @@ def test_will_recheck_access_handler_attribute(self, handler, will_recheck_acces
         request.user = self.mock_user
 
         render.handle_xblock_callback(request, str(course.id), usage_id, handler)
-        assert mock_get_block.call_count == 1
+        assert mock_get_block.call_count == 2
```

Comment:
This has increased because we renamed `get_block_for_descriptor_internal` to `get_block_for_descriptor`.